docs/package.json

```
@@ -91,7 +91,7 @@
     "lz-string": "^1.4.4",
     "markdown-to-jsx": "^7.1.7",
     "material-ui-popup-state": "^2.0.0",
-    "next": "^12.0.7",
+    "next": "12.1.5",
```

I had to lock it to `12.1.5`, because recently released `12.1.6` caused new issue: https://app.netlify.com/sites/material-ui/deploys/6270f5e015fd0e00085cd5b5#L172
We can resolve it later, it's unrelated to the one I'm trying to fix in this PR

---

I wonder if we should use ranges or pins.

```suggestion
    "next": "^12.0.7",
```

I mean, one or the other has no technical implications AFAIK, so maybe it doesn't matter. I'm saying no technical implications because the version is locked in `yarn.lock` and Renovate will try to update the version, regardless of the range used. More reading https://docs.renovatebot.com/dependency-pinning/#what-a-lock-file-doesnt-do-for-you.

---

In the link above, they suggest explicitly pinned version in package.json is somewhat better. I've never thought about it, but it does make sense.
But we only upgrade our dependencies one-by-one, so it shouldn't be that different.
I think in this case pinned version is justified and has an explicit intention of using this specific version of the package, so I would keep it pinned.